scripts/performance/report-as-a-pr-comment.js

```diff
@@ -146,6 +148,22 @@ function createMessage(
   })
   message += '\n</details>\n'
 
+  message += '\n\n<details>\n<summary>🧠 Memory Performance</summary>\n\n'
+  message +=
+    '| Action Name | Base Consumption Memory (bytes) | Local Consumption Memory (bytes) | 𝚫 |\n| --- | --- | --- | --- |\n'
```

Comment:
**💬 suggestion:** ‏This is a bit hard to read. You could write a function to format a markdown array, something like:

```
message += markdownArray({
  headers: ["Action name", ...],
  rows: differenceMemory.map((memoryActionPerformance) => {
    return [memoryActionNamePerformance.name, ...]
  })
})
```